diff --git a/app/desktopManager.js b/app/desktopManager.js
index 74bb937..4e5c259 100644
--- a/app/desktopManager.js
+++ b/app/desktopManager.js
@@ -430,14 +430,17 @@ var DesktopManager = class {
     }
 
     _setSelectionColor() {
-        this.selectColor = this._styleContext.get_background_color(Gtk.StateFlags.SELECTED);
+        this.selectColor = this._styleContext.get_color(Gtk.StateFlags.NORMAL);
         let style = `.desktop-icons-selected {
-            background-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.5);
+            background-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.3);
             border-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0);
         }
         .desktop-icons-keyboard-selected {
-            background-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.5);
-            border-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.8);
+            background-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.3);
+            border-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.5);
+        }
+        .file-item-hover {
+            background-color: rgba(${this.selectColor.red * 255},${this.selectColor.green * 255}, ${this.selectColor.blue * 255}, 0.1);
         }`;
         this._cssProviderSelection.load_from_data(style);
         Gtk.StyleContext.add_provider_for_screen(Gdk.Screen.get_default(), this._cssProviderSelection, 600);
diff --git a/app/stylesheet.css b/app/stylesheet.css
index f41c032..a761cc5 100644
--- a/app/stylesheet.css
+++ b/app/stylesheet.css
@@ -16,10 +16,6 @@
     border-color: rgba(0, 0, 0, 0);
 }
 
-.file-item-hover {
-   background-color: rgba(238, 238, 238, 0.2);
-}
-
 .not-found {
     color: rgb(255, 0, 0);
 }
